website/blog/2022/04/12/apisix-summit-asia-2022.md

```diff
@@ -0,0 +1,46 @@
+---
+title: "API Gateway Apache APISIX Summit ASIA 2022 Coming Soon"
+keywords:
+- Apache APISIX
+- Summit
+- API Gateway
+- API7.ai
+- Alibaba Cloud
+- Tencent Cloud
+- Tencent BlueKing
+- WPS
+- Snowball
+- Vmware Tanzu
+- Beeto
+- Excelliance Technology
+- Huya
+- UPYun
+description: The Apache APISIX community will organize the Apache APISIX Summit ASIA 2022 on May 20-21, 2022 via "Live Streaming".
+tags: [Events]
+---
+
+> The Apache APISIX community will organize the Apache APISIX Summit ASIA 2022 on May 20-21, 2022.
+
+<!--truncate-->
+
+![Summit Post](https://static.apiseven.com/202108/1649729812376-b05269c8-90b9-4d73-b97e-463beb351d1d.png)
+
+## Conference Introduction
+
+Since Apache APISIX was officially open sourced on June 6, 2019, it has been growing rapidly as a community. In just over two years, the number of global contributors has exceeded 400, and the number is still growing rapidly. During this time, the Apache APISIX community has also successively gained recognition from domestic and foreign developers.
+
+As an open source API gateway, Apache APISIX enables companies to quickly and securely process API and microservice traffic in scenarios such as gatewa ys, Kubernetes Ingress, and service grids. Now, hundreds of companies around the world use Apache APISIX to process business-critical traffic, including finance, Internet, manufacturing, retail, carriers, and so on.
+
+In order to help more companies, developers and open source enthusiasts better understand and use Apache APISIX, and to help community members better understand the latest progress and developments of Apache APISIX, the Apache APISIX community will organize the **Apache APISIX Summit ASIA 2022** online on May 20-21, 2022.
+
+Technical experts from API7.ai, AliCloud, Tencent Cloud, Tencent Blue Whale, WPS, Snowball, Vmware Tanzu, Beeto, Excelliance Technology, Huya, and UPYun will bring the most cutting-edge usecase and practical sharing of Apache APISIX.
+
+There are more important speakers: Fei Xiong (Partner of Matrix Partners China), Sheng Wu (Apache Member), Mark Shan (Chair of Tencent Open Source Alliance), Ming Wen (Apahce APISIX PMC Chair), Trista Pan (Co-founder & CTO of SphereEx), Jia Zhai (Co-founder of StreamNative), Yeliang Wang (Partner of API7.ai), Yubo Wang (Director of Developer Relations of AWS), Xiaosi Zhou (QingCloud Head of Container Division and KubeSphere Founder), to discuss open source infrastructure software, internationalization and ecological construction.
+
+![Speakers and Topics](https://static.apiseven.com/202108/1649733714127-75c87822-5c92-4152-aa20-c2224e75e751.jpg)
+
+## How to Participate
+
```

Comment:
how about add the slack?

Reply:
Then we can't collect registration information